rspec-scaffold.gemspec

```diff
@@ -19,9 +19,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
   spec.license       = 'MIT'
 
-  spec.required_ruby_version = '>= 1.9.3'
+  spec.required_ruby_version = '>= 2.0.0'
```

Comment:
Sadly, my colorizing 'luks' gem has a ruby v2 dep. For now I got around it by bumping rspec-scaffold's req to v2 as well.
How bad do we want to support versions before v2 in upcoming rspec-scaffold version? Seeing how latest version is at 2.4 and officially supported at 2.2, I am very inclined to drop the support.